src/NUnitEngine/nunit.engine.tests/Services/TestRunnerFactoryTests/Results/Net20TwoProjectExpectedRunnerResults.cs

```diff
@@ -60,11 +60,11 @@ private static RunnerResult GetProcessModelMultipleResult(DomainUsage domainUsag
                 case DomainUsage.Multiple:
                     return new RunnerResult
                     {
-                        TestRunner = typeof(AggregatingTestRunner),
+                        TestRunner = typeof(MultipleTestProcessRunner),
                         SubRunners = new[]
                         {
-                            new RunnerResult { TestRunner = typeof(MultipleTestProcessRunner) },
-                            new RunnerResult { TestRunner = typeof(MultipleTestProcessRunner) }
```

Comment:
This is a change to one process per project, rather than one process per assembly - which isn't what we want. However, if I'm understanding correctly, this wouldn't have worked before anyway - as an AggregatingTestRunner or subclass should never be the leaf of the tree.

Reply:
Not exactly. Since the project is not being expanded, I made the assumption for test purposes that it only includes one assembly. At some point, we should fake expansion of the project as in the MasterTestRunner tests and have projects with different numbers of assemblies.

Reply:
> At some point, we should fake expansion of the project as in the MasterTestRunner tests and have projects with different numbers of assemblies.

That's what my most recent commit does...I think this does indicate a bug here. But only a change in behaviour to something which was already broken, so I think we can fix it as a separate issue.